index.html

```diff
@@ -11,9 +11,58 @@
 </head>
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <a class="logo" href="index.html"><img src="/img/karma-logo.svg" alt="karma logo"></a>
+        <nav>
+            <ul class="navlist">
+                <li class="navitems"><a class="navlinks" href="#">Meet Karma</a> </li>
+                <li class="navitems"><a class="navlinks" href="#">Store</a> </li>
+                 <li class="navitems"><a class="navlinks" href="#">Blog</a> </li>
+                <li class="navitems"><a class="navlinks" href="#">Help</a> </li>
+                 <li class="navitems"><a class="navlinks" href="#">Login</a> </li>    
+            </ul>
+        </nav>
+    </header> 
+    <p>
+        <section class="hero">
```

Comment:
<section /> is a good choice for these containers (rather than article, because that's for content that could be shared / syndicated with other sites).